### DISCUSSION TITLE
Help: Account Suspended

### DISCUSSION THREAD

**Discussion by iam-baivab** [community/community#124146] [Other Feature Feedback, Questions, & Ideas]
### Account Suspended 

General

### Body

My GitHub account got suspended. Please help me to get back my account. I appeal for a Reinstatement.
It's been 6 days and I got no reply or support from GitHub. Please help me

**Comment by ebndev** [comment]
Hi @Again-Developer,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
